src/components/Dialog/Dialog.scss

```diff
@@ -148,7 +148,8 @@
 
   // Spacing between bottom buttons
   .ms-Dialog-action + .ms-Dialog-action {
-    margin: 0 0 0 16px;
+    margin: 0;
+    @include margin-right(16px);
```

Comment:
I think this should be `@include margin-left` to be consistent with the original values.